openmetadata-ui/src/main/resources/ui/src/pages/KPIPage/KPIMock.mock.ts

```diff
@@ -0,0 +1,444 @@
+/*
+ *  Copyright 2021 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+export const KPI_CHARTS = [
+  {
+    id: 'd2f093d4-0ca8-42b8-8721-1c2a59951b59',
+    name: 'dailyActiveUsers',
+    displayName: 'Daily active users on the platform',
+    fullyQualifiedName: 'dailyActiveUsers',
+    description: 'Display the number of users active.',
+    dataIndexType: 'web_analytic_user_activity_report_data_index',
+    dimensions: [
+      {
+        name: 'timestamp',
+        chartDataType: 'INT',
+      },
```

Comment:
Did we remove unnecessary data from these mock values to avoid adding unnecessary lines?

Reply:
@aniketkatkar97 , Here we need all the charts to filter out internally.